### ISSUE TITLE
False Positive: 'Attempted relative import beyond top-level package'

### ISSUE THREAD

**Issue by cm107** [PyCQA/pylint#3629]
### Steps to reproduce
1. Install the latest version of pylint (2.5.2)
```console
pip install pylint==2.5.2
```
2. Create a simple package folder with a pair of scripts that use a relative import.
```console=
mkdir pkg_folder
cd pkg_folder
touch __init__.py
code test0.py test1.py
```
test0.py
```python
class Test0:
    def __init__(self):
        pass
```
test1.py
```python
from .test0 import Test0

class Test1:
    def __init__(self):
        super().__init__()
        pass
```
Notice that pylint is giving you an 'Attempted relative import beyond top-level package' error.
![image](https://user-images.githubusercontent.com/48467759/82112111-d7c96a00-9784-11ea-949d-64b1de991f5f.png)
3. Verify that you are not going crazy: Install an older version of pylint that should work correctly.
```console
pip install pylint==2.4.4
```
![image](https://user-images.githubusercontent.com/48467759/82112184-8ff71280-9785-11ea-87eb-de7c0169917f.png)

The error went away.
In other words, there's something wrong with version 2.5.2.

### Current behavior
pylint is gives an 'Attempted relative import beyond top-level package' error when it is not warranted. This is a false positive.

### Expected behavior
This error should not be shown.

### pylint --version output
pylint 2.5.2
astroid 2.4.1
Python 3.7.3 (default, Jul 10 2019, 16:01:05) 
[GCC 7.4.0]

**Comment by Pierre-Sassoulas** [comment]
Hello, thank you for reporting this problem. When you report a bug please do not use screenshot as it's not searchable (not readable by screen readers and impossible to find with a web search), instead copy paste the text output :)
There is less chance that we'll have to handle duplicate issues that way.

**Comment by jnsnow** [comment]
Is this a duplicate of https://github.com/PyCQA/pylint/issues/3624 ?

**Comment by Pierre-Sassoulas** [comment]
Is does look like a duplicate, thank you @jnsnow
